.changeset/little-seals-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+'svelte': patch
+---
+
+feat: add ability to ignore warnings through compiler option
```

Comment:
Would it be possible to disable these warnings only for a sub-directory? My use case is a public facing app where I care about accessibility warnings, and a `routes/admin/*` section where I'm the sole user and where I'd like to disable these warnings.

<sub>(comment not related to this line, just here to create a thread)</sub>

Reply:
We're going to adjust the option to be a function instead that gets the warning and returns a boolean, so you can do that based on `warning.filename`